server-connection/src/main/java/org/sonarsource/sonarlint/core/serverconnection/storage/XodusServerIssueStore.java

```diff
@@ -457,6 +484,19 @@ private static void remove(String issueKey, @NotNull StoreTransaction txn) {
       });
   }
 
+  private static void removeTaint(String issueKey, @NotNull StoreTransaction txn) {
+    findUnique(txn, ISSUE_ENTITY_TYPE, KEY_PROPERTY_NAME, issueKey)
```

Comment:
Shouldn't it be TAINT_ISSUE_ENTITY_TYPE ?

Reply:
Good catch!